test/test-applications/integrations/Samples.XUnitTests/xunit.runner.json

```diff
@@ -0,0 +1,5 @@
+﻿{
+  "shadowCopy": false,
+  "diagnosticMessages": true,
+  "internalDiagnosticMessages": true
```

Comment:
Is it worth using one of the xunit method rename options in here, to make sure we handle that correctly? https://xunit.net/docs/configuration-files#methodDisplay

Reply:
good point, I'll check.

Reply:
Trying `method` in f91a299. I think if it everything passes I will revert to `classAndMethod` as the default one.

Reply:
fixed in 698a77a